pandas/core/generic.py

```diff
@@ -2740,17 +2740,20 @@ class      (index) object 'bird' 'bird' 'mammal' 'mammal'
     def to_latex(self, buf=None, columns=None, col_space=None, header=True,
                  index=True, na_rep='NaN', formatters=None, float_format=None,
                  sparsify=None, index_names=True, bold_rows=False,
-                 column_format=None, longtable=None, escape=None,
-                 encoding=None, decimal='.', multicolumn=None,
-                 multicolumn_format=None, multirow=None):
+                 column_format=None, longtable=None, caption=None,
+                 label=None, escape=None, encoding=None, decimal='.',
+                 multicolumn=None, multicolumn_format=None, multirow=None):
         r"""
-        Render an object to a LaTeX tabular environment table.
+        Render an object to a LaTeX tabular, longtable, or nested
```

Comment:
These summaries should fit on a single line (else they get truncated in the HTML docs).